pkgs/development/python-modules/sphinx-lv2-theme/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+  sphinx,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "sphinx-lv2-theme";
+  version = "1.4.2";
+  format = "setuptools";
```

Comment:
```suggestion
  pyproject = true;
```
Eventually, you will have to add `build-system = [ setuptools ];`.

Reply:
Indeed, I had to.